### PR TITLE
Laravel 7 support

### LINKED ISSUE #303: laravel 7 compatibility issue
I will like for this package to support laravel 7

### PR DESCRIPTION
**Pull request by stancl**
This adds Laravel 7 support.

Resolves #303

### CODE REVIEW

**Comment by stancl** [comment]
While all tests pass and the updated version constraints in composer.json should work with both Laravel 6 and Laravel 7, I can't merge this until @sebastianbergmann tags a new release of phpcov - https://github.com/sebastianbergmann/phpcov/pull/95#issuecomment-594753728

**Comment by stancl** [comment]
I'll try using `dev-master` for phpcov, thanks for the suggestion @morloderex.

File: Dockerfile [hunk 1]
```diff
@@ -4,27 +4,29 @@ LABEL maintainer="Samuel Štancl"
...
-    && apt-get install -y curl zip unzip git sqlite3 \
-    php7.2-fpm php7.2-cli \
-    php7.2-pgsql php7.2-sqlite3 php7.2-gd \
-    php7.2-curl php7.2-memcached \
-    php7.2-imap php7.2-mysql php7.2-mbstring \
-    php7.2-xml php7.2-zip php7.2-bcmath php7.2-soap \
-    php7.2-intl php7.2-readline php7.2-xdebug \
+RUN apt-get update
+
+RUN apt-get install -y software-properties-common
```
Comment (morloderex) [inline Dockerfile:8]: Not sure but maybe we can combine these with the `&&` operator?

Something like:
```
RUN apt-get update -y && apt-get install -y software-properties-common && add-apt-repository -y ppa:ondrej/php
```

Comment (stancl) [reply Dockerfile:8]: Good point

Comment (stancl) [reply Dockerfile:8]: This is related to the line below too though. And I don't want to combine all 3, so I'll keep this the way it is. I cleaned up a different part of the file.

**Review by morloderex — APPROVED** [review]

**Comment by stancl** [comment]
For some reason the testbench version constraint doesn't work when `composer require laravel/framework:^6.0` is used?

https://travis-ci.com/stancl/tenancy/jobs/294122914#L2232

File: .travis.yml [hunk 2]
```diff
@@ -1,20 +1,20 @@
-# env:
-#   - LARAVEL_VERSION="^6.0" TESTBENCH_VERSION="~4.0" REDIS_DRIVER=phpredis
+env:
+  - LARAVEL_VERSION="^7.0"
```
Comment (morloderex) [inline .travis.yml:2]:
```
env:
  - LARAVEL_VERSION="^6.0" TESTBENCH_VERSION="~4.0"
  - LARAVEL_VERSION="^7.0" TESTBENCH_VERSION="~5.0"
```

Might work (combined with the composer reqiure stuff down below) to resolve the issues related to https://github.com/stancl/tenancy/pull/304#issuecomment-594857474

Comment (stancl) [reply .travis.yml:2]: I think it would work, but I wonder why the current solution doesn't work

**Comment by stancl** [comment]
I might slightly delay this, but (in the next few days anyway) I'd like to make this part of the 2.3.0 release, now that #237 can be merged, which is also more semantically correct than making this a patch release.

**Comment by stancl** [comment]
If this https://github.com/stancl/tenancy/issues/301#issuecomment-599201192 solves #301 then it should already work in L7 with the different installation steps. I'll have to add it to the L6 installation steps.

File: .travis.yml (hunk 2 above)
Comment (stancl) [reply .travis.yml:2]: Seems like this just started working?

**Pull request merged**